packages/valory/skills/price_estimation_abci/behaviours.py

```diff
@@ -303,6 +299,24 @@ def get_service_data(self) -> Dict:
             "unit": f"{price_api.currency_id}:{price_api.convert_id}",
         }
 
+    def get_data_signature(self, data: Dict) -> Generator[None, None, Tuple[str, str]]:
+        """Get signature for the data."""
+        data = copy.deepcopy(data)
+
+        data.pop("agent_address", None)  # agent address is unique, need to remove it
+        data.pop("data_source", None)  # data_source can be unique, need to remove it
```

Comment:
So if the data source is unique, doesn't this mean that values in data can be unique?

Reply:
I believe yes, I am not sure what exactly @solarw meant with this comment. 
This part - including the comment - has just been moved from the `SignServiceDataHashBehaviour` which was deleted.

Reply:
Removed on abf459bb5fdef5101eb2d4fa9db13bde6428c4dc.